src/Propel/Generator/Model/Schema.php

```diff
@@ -105,7 +105,7 @@ public function setGeneratorConfig(GeneratorConfigInterface $generatorConfig)
      *
      * @return \Propel\Generator\Config\GeneratorConfigInterface
      */
-    public function getGeneratorConfig()
+    public function getGeneratorConfig(): GeneratorConfigInterface
```

Comment:
Many errors comes from here.
```
TypeError: Propel\Generator\Model\Schema::getGeneratorConfig(): Return value must be of type Propel\Generator\Config\GeneratorConfigInterface, null returned
```

What should we do with this one?

Reply:
Make those nullable, see comments above

Reply:
So the global strategy for this PR is to find liars Docblocks?

Reply:
well, they are found as byproduct of getting the fatal errors and failing tests resolved I guess.

Reply:
in some cases, where they are directly used and expected to be not nullable, it can also make sense to add a has() method to check for existance. then you can has() before get() instead of nullable get() and having to check for return value.
So this is not quite as simple if we want to create a good API moving forward, but the quick way sure would be the nullable fixes.